wait/host_port_test.go

```diff
@@ -454,11 +455,15 @@ func TestHostPortStrategySucceedsGivenShellIsNotInstalled(t *testing.T) {
 		WithStartupTimeout(5 * time.Second).
 		WithPollInterval(100 * time.Millisecond)
 
-	oldWriter := log.Default().Writer()
-	var buf bytes.Buffer
-	log.Default().SetOutput(&buf)
+	oldLogger := log.Default()
+
+	buf := &bytes.Buffer{}
+	logger := standardlog.New(buf, "test", standardlog.LstdFlags)
+	logger.SetOutput(buf)
```

Comment:
suggestion: you can just declare and no need to set the output that was done by the New, more below.
```suggestion
	var buf bytes.Buffer
	logger := standardlog.New(&buf, "test", standardlog.LstdFlags)
```

Reply:
Is this right? the `buf` variable is already a pointer, I don't think we need the extra `&`?

Reply:
Just pushed the fix